Frameworks/PSMTabBar/Styles/PSMSequelProTabStyle.m

```diff
@@ -399,6 +401,10 @@ - (NSAttributedString *)attributedStringValueForTabCell:(PSMTabBarCell *)cell
 // Step 1
 - (void)drawTabBar:(PSMTabBarControl *)bar inRect:(NSRect)rect
 {
+	if (@available(*, macOS 10.14)) {
```

Comment:
I can't approve this as we are not using this available selector for backwards compatibility...

Reply:
Which minimal version must be supported?

Reply:
Found: 10.8 SDK
https://github.com/sequelpro/sequelpro/blob/df10e204431c40967aa4d3377a5c5334f39b7fa9/Source/SPCompatibility.h#L48
And whole file is interesting.. will try to do something with this in nearest days.

Reply:
Nope, it's not about minimum version, it's about method you need to use instead of this macro. :)

Reply:
We already have checks for Dark Mode, try to reuse that please.

Reply:
Now I see, will try. That time I haven't looked the code deeply, and yep, my ObjC knowledge is still about zero :)

Reply:
For 1.2:
The minimum development version (SDK, Xcode) to support is 10.8.
The minimum runtime version to support is 10.6.

Reply:
Changed to old style in 1eb8791

Reply:
Did this change again since https://github.com/sequelpro/sequelpro/commit/1eb8791a20ddf34574de70ea9094064247f2c9dd ?

Reply:
Sorry, now syncing my master with https://github.com/Sequel-Ace/Sequel-Ace fork, looks like they decided to drop old XCode versions, cancelling my pull request (anyway looks like it will never be accepted, more than a year passed).

Reply:
I was just about to accept it and merge, all righty.

Reply:
Fortune is strange, just took my hands on it few hours ago too :) I can try to revert to last significant commit in another branch and to create new request if this is still helpful.

Reply:
Nah, I'm just poking around here, still don§t have full permissions to do a release or anything, so this project will probably die soon. Thanks tho!

Reply:
Thank you too. Looks like you the only one who reviewed my commits here! :)

Reply:
@Kaspik Love to see someone reviewing Sequel Pro PRs again! Honestly though, if the Sequel Pro project is gonna die soon, we'd love to have more help on the Sequel Ace project. Anyone involved in Sequel Pro is definitely very welcome on the Ace train. Sequel Ace is the realization that Sequel Pro was dying/dead and the best thing to do was to stop complaining about it and start forking it.

Reply:
@Jason-Morcos Hey! Would you mind sending me an invite to the org and repo so I don’t have to fork it as well for PRs? I’ll check it out!

Also, is there some Slack workspce or anything?

Reply:
Invited, @Kaspik!

No Slack yet, was thinking about that though! Probably will make one today